### PR TITLE
CDOT CV Manager Theme Generator

### PR DESCRIPTION
**Pull request by Michael7371**
Added the required changes to the keycloakify-starter repo for the CDOT CV Manager theme.

### CODE REVIEW

**Review by drewjj — APPROVED** [review]
Looks good to me.

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -9,22 +9,30 @@ on:
 
 jobs:
 
-  test:
+  build:
```
Comment (payneBrandon) [inline .github/workflows/ci.yaml:12]: I hadn't dug into this yaml file previously, but it looks like we're triggering on push and pull requests to the main branch. Does that mean any PR will trigger the full CI? It looks like we have a build and release below, that may not be what we want

**Review by payneBrandon — CHANGES_REQUESTED** [review]
Just the one question about the CI

**Review by payneBrandon — APPROVED** [review]
lgtm!

**Pull request merged**